src/hooks/historikk/useVeilederHistorikk.ts

```diff
@@ -0,0 +1,45 @@
+import { HistorikkEvent } from "@/data/historikk/types/historikkTypes";
+import {
+  useVeilederHistorikk,
+  VeilederHistorikkDTO,
+} from "@/data/veilederbrukerknytning/useGetVeilederBrukerKnytning";
+import { BrukerinfoDTO } from "@/data/navbruker/types/BrukerinfoDTO";
+import { useBrukerinfoQuery } from "@/data/navbruker/navbrukerQueryHooks";
+
+function createHistorikkEventsFromVeilederHistorikk(
+  veilederHistorikkDTO: VeilederHistorikkDTO[],
+  person: BrukerinfoDTO
+): HistorikkEvent[] {
+  return veilederHistorikkDTO.map((value) => ({
+    opprettetAv: value.tildeltAv,
+    tekst: `${value.tildeltAv} satt ${value.tildeltVeileder} i ${value.tildeltEnhet} som veileder`,
+    tidspunkt: value.tildeltDato,
+    kilde: "VEILEDER",
+  }));
+}
+
+interface VeilederHistorikk {
+  isVeilederHistorikkLoading: boolean;
+  isVeilederHistorikkError: boolean;
+  veilederHistorikk: HistorikkEvent[];
```

Comment:
Hvis vi forenkler navngivningen her kan det brukes på denne måten hvor hooken brukes:
```
const veilederHistorikk = useVeilederBrukerHistorikk();
veilederHistorikk.events
veilederHistorikk.isLoading
veilederHistorikk.isError
```
```suggestion
  isLoading: boolean;
  isError: boolean;
  events: HistorikkEvent[];
```
Tror dette kan gjøre koden litt lettere å lese, og litt ryddigere 😊

Reply:
Følger vel bare mønsteret jeg satt opp til her i den forrige PRen min 🙃 Men enig i at vi bare kan endre det til dette 👍🏼